modules/claims_api/lib/claims_api/v2/disability_compensation_evss_mapper.rb

```diff
@@ -0,0 +1,121 @@
+        current_mailing_address
+        direct_deposit
+        disabilities
+        service_information
+        standard_claim
+        veteran_meta
+      end
+
+      def change_of_address
+        return if @data[:changeOfAddress].blank?
+
+        @evss_claim[:veteran] ||= {}
+        @evss_claim[:veteran][:changeOfAddress] = @data[:changeOfAddress]
+        @evss_claim[:veteran][:changeOfAddress].merge!({
+                                                         addressLine1: @data[:changeOfAddress][:numberAndStreet],
+                                                         addressLines2: @data[:changeOfAddress][:apartmentOrUnitNumber],
+                                                         type: 'DOMESTIC'
+                                                       })
+        @evss_claim[:veteran][:changeOfAddress][:addressChangeType] = @data.dig(:changeOfAddress, :typeOfAddressChange)
+        @evss_claim[:veteran][:changeOfAddress][:beginningDate] = @data.dig(:changeOfAddress, :dates, :beginningDate)
+        @evss_claim[:veteran][:changeOfAddress][:endingDate] = @data.dig(:changeOfAddress, :dates, :endingDate)
+        @evss_claim[:veteran][:changeOfAddress].except!(:numberAndStreet, :apartmentOrUnitNumber, :typeOfAddressChange,
+                                                        :dates)
+      end
+
+      def current_mailing_address
+        addr = @data.dig(:veteranIdentification, :mailingAddress) || {}
+        @evss_claim[:veteran] ||= {}
+        @evss_claim[:veteran][:currentMailingAddress] = addr
+        @evss_claim[:veteran][:currentMailingAddress].merge!({
+                                                               addressLine1: addr[:numberAndStreet],
+                                                               addressLines2: addr[:apartmentOrUnitNumber],
+                                                               type: 'DOMESTIC'
+                                                             })
+        @evss_claim[:veteran][:currentMailingAddress].except!(:numberAndStreet, :apartmentOrUnitNumber)
+      end
+
+      def direct_deposit
+        return if @data[:directDeposit].empty?
+
+        @evss_claim[:directDeposit] = @data[:directDeposit]
+        @evss_claim[:directDeposit][:bankName] = @data[:directDeposit][:financialInstitutionName]
+        @evss_claim[:directDeposit].except!(:financialInstitutionName, :noAccount)
+      end
+
+      def disabilities
+        @evss_claim[:disabilities] = @data[:disabilities].map do |disability|
+          disability[:approximateBeginDate] = map_date_to_obj disability[:approximateDate]
+          disability[:secondaryDisabilities] = disability[:secondaryDisabilities].map do |secondary|
+            secondary[:approximateBeginDate] = map_date_to_obj secondary[:approximateDate]
+            secondary.except(:exposureOrEventOrInjury, :approximateDate)
+          end
+
+          disability[:specialIssues] = 'PACT' if disability[:isRelatedToToxicExposure]
```

Comment:
I'm not 💯 on this one, sometimes they complain about it, sometimes they don't, so this may need to be something else (I.E. `['PACT']` or `{code: 'PACT', name: 'PACT'}`)